springfield/firefox/templates/firefox/all/download.html

```diff
@@ -6,7 +6,7 @@
 
 {% from "macros.html" import google_play_button, apple_app_store_button, ms_store_button with context %}
 
-{% set icon_download ='<span class="mzp-c-button-icon-end"><svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 16 16" width="16" height="16" fill="currentColor"><path d="M8 13c.2 0 .4-.1.5-.2l4.4-4.4-1.1-1.1-3.1 3.1V1H7.2v9.4L4.1 7.3l-1 1.1 4.4 4.4c.1.1.3.2.5.2z" /><path d="M13.5 12v2c0 .3-.2.5-.5.5H3c-.3 0-.5-.2-.5-.5v-2H1v2c0 1.1.9 2 2 2h10c1.1 0 2-.9 2-2v-2h-1.5z" /></svg></span>' %}
+{% set icon_download ='<span class="mzp-c-button-icon-end"><svg height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><clipPath id="a"><path d="m0 0h16v16h-16z"/></clipPath><g clip-path="url(#a)" fill="currentColor"><path d="m7.2507 0v10.21l-3.72-3.72-1.06 1.06 5.53 5.53 5.53-5.53-1.06-1.06-3.72 3.72v-10.21z"/><path d="m16 14.5h-16v1.5h16z"/></g></svg></span>' %}
```

Comment:
```suggestion
{% set icon_download ='<span class="mzp-c-button-icon-end"><svg height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><clipPath id="a"><path d="m0 0h16v16h-16z"/></clipPath><g clip-path="url(#a)" fill="currentColor"><path d="m7.2507 0v10.21l-3.72-3.72-1.06 1.06 5.53 5.53 5.53-5.53-1.06-1.06-3.72 3.72v-10.21z"/><path d="m16 14.5h-16v1.5h16z"/></g></svg></span>' %}
```

seeing that here the empty fill is not used.

(Just for consistency; or to be sure why we have the attribs in.)